### DISCUSSION TITLE
Unable to start server using ./gradlew serverRun

### DISCUSSION THREAD

**Discussion by Afreen97** [TEAMMATES/teammates#12689] [Help requests] [answered]
Hello,
 I am willing to contribute.I am cloned the repo and ran the following commands-
1. docker compose up -d
2. ./gradlew createConfigs.

I am trying to start the server via ./gradlew serverRun.But I am encountering an error -

> Task :compileJava FAILED
/home/afreen/Documents/OpenSource/teammates/src/main/java/teammates/common/util/Logger.java:11: error: cannot access javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletRequest;
                         ^
  bad class file: /home/afreen/.gradle/caches/modules-2/files-2.1/org.eclipse.jetty.toolchain/jetty-servlet-api/4.0.6/959c5d83d08f5cddf56caff749e48b735193191b/jetty-servlet-api-4.0.6.jar(javax/servlet/http/HttpServletRequest.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.

FAILURE: Build failed with an exception.

**Steps to reproduce**
Run  ./gradlew serverRun after cloning repo and running createConfigs command.

Can anyone suggest an way out?

**Comment by domlimm** [comment, marked as answer]
@Afreen97 May I ask what's your JDK version? You can check via this command in your terminal at anywhere `java --version`.

**Comment by Afreen97** [comment]
@domlimm I had jdk 1.8 installed.I went through the documentation and I am now using jdk 11.Now it is working fine.Thanks for help!

**Reply by domlimm** [reply]
@Afreen97 you're most welcome. Let us know if you need any other help. Will close this now.
